cmd/it.go

```diff
@@ -108,8 +111,8 @@ func tunnelMultiple(confs []tunnelConf) {
 	}
 	var wg sync.WaitGroup
 	wg.Add(len(tunnelConfigs))
-	for _, conf := range tunnelConfigs {
-		go tunnel.StartReverseTunnel(conf, &wg)
+	for i := 0; i < len(tunnelConfigs); i++ {
```

Comment:
Why change this?

Reply:
The linter was complaining about big struct copying. I can change it back and hide that warning